bindings/go/context.go

```diff
@@ -9,10 +9,6 @@ package ucp
 // #include <ucs/type/status.h>
 import "C"
 
-import (
-	"fmt"
-)
```

Comment:
why removed?

Reply:
Error now propagates like `NewUcxError(status)`. So it would be possible to get status and message from it.